### PR TITLE
Reports via Email

### LINKED ISSUE #707: Repot via Email

### PR DESCRIPTION
**Pull request by axel1200**
Fixes #707

### CODE REVIEW

**Comment by 1-alex98** [comment]
What do we do with that?

**Comment by 1-alex98** [comment]
Do we want this?

**Comment by micheljung** [comment]
We certainly want the event publishing refactoring, but I forgot what the status of the "send email" thing is :) Too much to look through atm :P

File: src/main/java/com/faforever/client/reporting/SupportServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      "username: {0}\n" +
+      "clientVersion: {1}\n" +
+      "supportType: {2}\n" +
+      "system: {3}\n" +
+      "\n" +
+      "error message:\n" +
+      "{4}\n" +
+      "\n" +
+      "lastLogStatements:\n" +
+      "{5}";
```
Comment (Geosearchef) [inline src/main/java/com/faforever/client/reporting/SupportServiceImpl.java:51]: Are we sure that we wanna put this in code instead of a resource file?

Comment (1-alex98) [reply src/main/java/com/faforever/client/reporting/SupportServiceImpl.java:51]: Yeah we wanna put it in an resource file

**Comment by 1-alex98** [comment]
@micheljung I need to know if we want the email thingy

**Comment by micheljung** [comment]
The email thingy doesn't really work as we would need it to, so no for now?

**Pull request closed without merging**